applications/GeoMechanicsApplication/custom_conditions/T_microclimate_flux_condition.cpp

```diff
@@ -169,14 +169,12 @@ void GeoTMicroClimateFluxCondition<TDim, TNumNodes>::CalculateAndAddRHS(
         outer_prod(rN, rN) * IntegrationCoefficient};
     auto temporary_vector =
         array_1d<double, TNumNodes>{prod(temporary_matrix, rRightHandSideFluxes)};
-    GeoElementUtilities::AssemblePBlockVector<0, TNumNodes>(
-        rRightHandSideVector, temporary_vector);
+    rRightHandSideVector += temporary_vector;
```

Comment:
Do we still need to create the temporary_vector variable or is the following enough:
rRightHandSideVector += prod(temporary_matrix, rRightHandSideFluxes);

The same happens on 176-177

Reply:
Good catch. Indeed, we can eliminate variable `temporary_vector`. Done.